### ISSUE TITLE
Line is over dot on piggy bank scene and others

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/number-line-integers#88]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/QA/issues/468. May be related to https://github.com/phetsims/number-line-integers/issues/83. This did not occur in rc 2.
On the piggy bank scene the line that leads from the piggy bank to the number line is now layered over the dot on the number line. This looks very off.

**Visuals**
![layeroverdot](https://user-images.githubusercontent.com/41024075/73085903-c59d8600-3e8c-11ea-8884-874bc54e59a6.png)

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Number Line: Integers‬
URL: https://phet-dev.colorado.edu/html/number-line-integers/1.0.0-rc.3/phet/number-line-integers_en_phet.html
Version: 1.0.0-rc.3 2020-01-24 00:21:22 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36
Language: en-US
Window: 1536x722
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by KatieWoe** [comment]
Same sort of layering issue with absolute value on the elevation scene. Not in rc 2.
![sameissue](https://user-images.githubusercontent.com/41024075/73091146-05696b00-3e97-11ea-9e3d-5eba16000fc3.png)
EDIT: This occurred in rc 2 and is likely not an issue.

**Comment by KatieWoe** [comment]
![labels](https://user-images.githubusercontent.com/41024075/73091212-2d58ce80-3e97-11ea-8aef-d20cdacdf683.png)

**Comment by KatieWoe** [comment]
![moreover](https://user-images.githubusercontent.com/41024075/73092551-149de800-3e9a-11ea-8333-6e0dd1e90037.png)

**Comment by KatieWoe** [comment]
Odd layering that occurred in rc2 as well:
![oddlayering](https://user-images.githubusercontent.com/41024075/73106028-a536f100-3eb7-11ea-9b9b-c642b2126de9.png)
Does not need string test, just easier to reproduce with.

**Comment by jbphet** [comment]
The bank scene behavior was due to the re-layering that was done for #83.  I revised things such that the point controllers are in front of the number line(s) in the "Elevation" and "Temperature" scenes, but behind it in the "Bank" scene.  This seems to work, I'll leave it to QA to make sure there weren't any undesired consequences.

The combo box layering was due to a different issue, and has also been fixed.
